docs/API/core/app-actions.md

```diff
@@ -326,6 +326,42 @@ Fliplet.App.Actions.remove('confirm-booking').then(function () {
 
 ---
 
+## Testing and debugging an action
+
+Once you have created an app action and added the code you want to run to the onRemoteExecution() function you should test it and see if it running as expected. Note that you cannot use developer tools to view your console.log inside the onRemoteExecution() function as this is running on the server side. Here is the general steps to test an app action: 
+
+1. Create the app action 
+2. Add code to onRemoteExecution() that you want to run in the app action. 
+3. Call the app action on demand and log the returned data to see if it working as expected. 
+4. If there are multiple steps in your code then move the `return Promise.resolve()` to the point where you want to see the data available. 
```

Comment:
I don't understand what this is supposed to do

Reply:
Users are not sure how to test their app action code. This is general steps to debug step by step. People are creating and setting up the actions but have no idea to verify it is working as expected.